README.md

```diff
@@ -26,7 +26,6 @@ Annotating any case class with `mappable` will generate a companion object (or e
 For details check out [MappableTest.scala](examples/src/test/scala/scala/meta/serialiser/MappableTest.scala)
 
 ## current limitations (a.k.a. TODOs)
```

Comment:
I find another limitations from comments. Maybe we can add `support for multiple constructor params lists` to this.